t/ids_from_pages.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+describe 'Wikidata#ids_from_pages' do
+  around { |test| VCR.use_cassette(titles.join, &test) }
```

Comment:
You probably want to join this with a separator that can't exist in a title, otherwise it won't distinguish between `['Foo', 'Bar']` and `['FooBar']`